### ISSUE TITLE
Bug: Query double attributes with integer (ex: greaterThan 0)

### ISSUE THREAD

**Issue by stnguyen90** [appwrite/appwrite#4340]
### Discussed in https://github.com/appwrite/appwrite/discussions/4327

<div type='discussions-op-text'>

<sup>Originally posted by **alexbacanu** October  7, 2022</sup>
I have this attribute:
![image](https://user-images.githubusercontent.com/12121946/194593304-016473d1-b3ec-4da7-a208-5da4f541b22c.png)
Attribute ID: productStars
Type: Double

It's an attribute that will be a float between 0 and 5 (can be 3.5, 4.7, etc)

When I try to query using the [Web SDK](https://appwrite.io/docs/client/databases) I get the following error:
```
AppwriteException: Query not valid: Query type does not match expected: double
    at Client.<anonymous> (file:///~/node_modules/.pnpm/appwrite@10.1.0/node_modules/appwrite/dist/esm/sdk.js:385:27)
    at Generator.next (<anonymous>)
    at fulfilled (file:///~/node_modules/.pnpm/appwrite@10.1.0/node_modules/appwrite/dist/esm/sdk.js:22:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
  code: 400,
  type: 'general_argument_invalid',
  response: {
    message: '**Query not valid: Query type does not match expected: double**',
    code: 400,
    type: 'general_argument_invalid',
    version: '1.0.2'
  }
}
```

Query example:
```
const promise = databases.listDocuments(databaseID, collectionID, [
    Query.greaterThanEqual("productStars", 0),
    Query.limit(60),
  ]);
```

In JavaScript I cannot write 0.0. How can I query for objects greater than 0 (number type) on a double attribute?
While writing this I realized I can query by -0.01 but that doesn't look right, I use these in an url and it looks wrong with minReviews=-0.01 in the url string.</div>

**Comment by stnguyen90** [comment]
FYI, this is a problem with passing any integer for a float attribute.

**Comment by stnguyen90** [comment]
@abnegate, this isn't in appwrite yet, right? We need a utopia-php/database release and a version bump in appwrite?

**Comment by abnegate** [comment]
@stnguyen90 Correct, it won't be included in 1.2.1 as the database release includes other changes we don't want in Appwrite yet 👍
